docs/docs/getting-started.jp.md

```diff
@@ -0,0 +1,116 @@
+---
+id: getting-started-ja
```

Comment:
Overall this looks great!

The only thing I see being an issue is the locale id. Correct me if I'm wrong, but you probably want `ja-JP`. We already have `zh-CN` and #3240 adds `ko-KR`. Let's update and make sure the filenames, ids, and links are correct.

I'm far from an expert here, I'd just like to make sure we're consistent.